alpa/parallel_method.py

```diff
@@ -166,6 +167,8 @@ class PipeshardParallel(ParallelMethod):
         stage_option: Options of grouping layers into pipeline stages.
           Possible choices are {"uniform", "auto", alpa.AutoStageOption,
                                  alpa.ManualStageOption}
+        input_sharding: Options of input sharding specs for each stage.
```

Comment:
```suggestion
        stage_input_sharing: Options of input sharding specs for each stage.
```

Because this is not the sharding of global vars, let us use `stage_input_sharding` to avoid confusion.

Reply:
Good point. Renamed.